src/saveload/oldloader_sl.cpp

```diff
@@ -710,7 +710,7 @@ static bool LoadOldGood(LoadgameState *ls, int num)
 	SB(ge->status, GoodsEntry::GES_ACCEPTANCE, 1, HasBit(_waiting_acceptance, 15));
 	SB(ge->status, GoodsEntry::GES_RATING, 1, _cargo_source != 0xFF);
 	if (GB(_waiting_acceptance, 0, 12) != 0 && CargoPacket::CanAllocateItem()) {
-		ge->cargo.Append(new CargoPacket(GB(_waiting_acceptance, 0, 12), _cargo_periods, (_cargo_source == 0xFF) ? INVALID_STATION : _cargo_source, INVALID_STATION, 0),
+		ge->cargo.Append(new CargoPacket(GB(_waiting_acceptance, 0, 12), _cargo_periods, (_cargo_source == 0xFF) ? INVALID_STATION : _cargo_source, INVALID_TILE, 0),
```

Comment:
With the change to INVALID_TILE...is this built upon #11281? Or is this a mistake?

Reply:
`next_station` is removed from the ctor, and `feeder_share` is no longer optional. So the diff shows a bit weird, and the original was wrong. It set the `source_xy` to zero, instead of `INVALID_TILE`.

This code was really old, and really annoying to understand.